ddd-koans/Value Object Pt. 2 & Entity/lesson-info.yaml

```diff
@@ -0,0 +1,5 @@
+type: framework
+content:
+- Introduction
+- Length and tempo
+- Draft
```

Comment:
Possibly in the next task, it would be good to summarize the previous task and explain that `lengthIn` is a VO behaviour so the yet another property called  Behavior-Rich